pkg/minikube/download/image.go

```diff
@@ -46,50 +44,55 @@ var (
 )
 
 // imagePathInCache returns path in local cache directory
-func imagePathInCache(img string) string {
+func imagePathInMinikubeCache(img string) string {
 	f := filepath.Join(detect.KICCacheDir(), path.Base(img)+".tar")
 	f = localpath.SanitizeCacheDir(f)
 	return f
 }
 
-// ImageExistsInCache if img exist in local cache directory
-func ImageExistsInCache(img string) bool {
-	f := imagePathInCache(img)
+// ImageExistsInCache if img exist in local minikube cache directory
+func ImageExistsInMinikubeCache(img string) bool {
+	f := imagePathInMinikubeCache(img)
 
 	// Check if image exists locally
-	klog.Infof("Checking for %s in local cache directory", img)
+	klog.Infof("Checking for %s in local minikube cache directory", img)
 	if st, err := os.Stat(f); err == nil {
 		if st.Size() > 0 {
-			klog.Infof("Found %s in local cache directory, skipping pull", img)
+			klog.Infof("Found %s in local minikube cache directory, skipping pull", img)
 			return true
 		}
 	}
 	// Else, pull it
 	return false
 }
 
-var checkImageExistsInCache = ImageExistsInCache
+var checkImageExistsInMinikubeCache = ImageExistsInMinikubeCache
 
-// ImageExistsInDaemon if img exist in local docker daemon
-func ImageExistsInDaemon(img string) bool {
-	// Check if image exists locally
-	klog.Infof("Checking for %s in local docker daemon", img)
-	cmd := exec.Command("docker", "images", "--format", "{{.Repository}}:{{.Tag}}@{{.Digest}}")
-	if output, err := cmd.Output(); err == nil {
-		if strings.Contains(string(output), image.TrimDockerIO(img)) {
-			klog.Infof("Found %s in local docker daemon, skipping pull", img)
-			return true
-		}
+// ImageExistsInKICDriver
+// checks for the specified image in the container engine's local cache
+func ImageExistsInKicDriver(ociBin, img string) bool {
+	klog.Infof("Checking for %s in local KICdriver's cache", img)
+	inCache := oci.IsImageInCache(ociBin, img)
+	if inCache {
+		klog.Infof("Found %s in local KICdriver's cache, skipping pull", img)
+		return true
 	}
-	// Else, pull it
 	return false
```

Comment:
The `return true` from above can be removed and this can just be `return inCache`.

Reply:
should we keep logging here, or just blindly return inCache?